Program.cs

```diff
@@ -1,12 +1,261 @@
+
+            Console.WriteLine($"Student - {student.PersonSurname}" +
+                $" {student.PersonName}," +
+                $" {student.PersonAge} years old.");
+            Console.WriteLine($"Ability: {student.StudentAbility}. Assesment: {student.StudentAssesment}.");
+
+            var teacher = new Teacher();
+            teacher.SetSurname("Ivanov");
+            teacher.SetName("Ivan");
+            teacher.SetAge(41);
+
+            teacher.SetTeacherTemperament("sanguine");
+            teacher.SetTeacherSubject("Mathematicks");
+
+            Console.WriteLine($"Teacher - {teacher.PersonSurname} " +
+                $"{teacher.PersonName}, " +
+                $"{teacher.PersonAge} years old. ");
+            Console.WriteLine($"Temperament: {teacher.TeacherTemperament}. Subject: {teacher.TeacherSubject}.");
+
+            //Exercise 3;
+            Console.WriteLine("Exercise 3.");
+
+            var personSecond = new Person();
+            personSecond.PersonSurname = "Datchin";
+            personSecond.PersonName = "Valery";
+            personSecond.PersonAge = 43;
+
+            Console.WriteLine($"{personSecond.GetSurname()} {personSecond.GetName()}," +
+            $" {personSecond.GetAge()} y.o.");
+
+            ////Exercise 4;
+            Console.WriteLine("Exercise 4.");
+
+            var myPerson = new Person();
+            myPerson.SetSurname("Fleming");
+            myPerson.SetName("Lise");
+            myPerson.SetAge(33);
+            myPerson.Print();
+
+            var myStudent = new Student();
+            myStudent.SetSurname("Koval");
+            myStudent.SetName("Irina");
+            myStudent.SetAge(19);
+            myStudent.Print();
+
+            var myTeacher = new Teacher();
+            myTeacher.SetSurname("Ivanov");
+            myTeacher.SetName("Ivan");
+            myTeacher.SetAge(41);
+            myTeacher.Print();
+
+            ////Exercise 5;
+            Console.WriteLine("Exercise 5.");
+
+            var classroom = new Classroom();
+            classroom.AddPersons();
+            classroom.Print();
+            classroom.RemovePersons();
+            classroom.ClearPersons();
+        }
+    }
+}
 
 namespace Exercises
 {
-    public static class Program
+    public class Person : IPrintable
     {
-        private static void Main()
+        private string _personSurname;
+        private string _personName;
+        private int _personAge;
+
+        public string PersonSurname
+        {
+            set { SetSurname(_personSurname = value); }
+            get { return GetSurname(); }
+        }
+
+        public string PersonName
+        {
+            set { SetName(_personName = value); }
+            get { return GetName(); }
+        }
+
+        public int PersonAge
+        {
+            set
+            {
+                SetAge(_personAge = value);
+            }
+            get
+            {
+                return GetAge();
+            }
+        }
```

Comment:
Such a record is possible, but it makes no sense to return it to a getter via a method.

It is also much less readable on its own.
Use auto properties or methods.